app/views/pages/home.html.erb

```diff
@@ -69,3 +69,21 @@
     </div> -->
   </div>
 </section>
+<footer role="“contentinfo”">
```

Comment:
Should this footer be abstracted into a partial and put into every page except for the workspace?